tox.ini

```diff
@@ -19,29 +19,32 @@ envlist =
     docs
 
 [testenv]
-commands = pytest --lsof -ra {posargs:testing}
+commands = {env:TOX_COVERAGE_RUN:} pytest --lsof -ra {posargs:testing}
 passenv = USER USERNAME
 deps =
     hypothesis>=3.5.2
     nose
     mock
     requests
+    {env:TOX_EXTRA_DEPS:}
 
 [testenv:py26]
 # pinning mock to last supported version for python 2.6
 deps =
     hypothesis<3.0
     nose
     mock<1.1
+    {env:TOX_EXTRA_DEPS:}
 
 [testenv:py27-subprocess]
 changedir = .
 deps =
     pytest-xdist>=1.13
     mock
     nose
+    {env:TOX_EXTRA_DEPS:}
 commands =
-    pytest -n3 -ra --runpytest=subprocess {posargs:testing}
+    {env:TOX_COVERAGE_RUN:} pytest -n3 -ra --runpytest=subprocess {posargs:testing}
```

Comment:
I'm seeing this warning from tox:

```
WARNING:test command found but not installed in testenv
  cmd: /usr/bin/env
  env: /home/travis/build/pytest-dev/pytest/.tox/py27
```

Do we have to whitelist `env`?

Reply:
Couldn't we just use [pytest-cov](https://github.com/pytest-dev/pytest-cov) instead?

Reply:
pytest-cov does incomplete reports due to late plugin startup, its a longstanding pytest issue